### ISSUE TITLE
[Table] 编辑单元格无效,控制台报错

### ISSUE THREAD

**Issue by xiufj** [Tencent/tdesign-vue-next#3575]
### tdesign-vue-next 版本

1.6.7

### 重现链接

https://tdesign.tencent.com/vue-next/components/table#%E5%8F%AF%E7%BC%96%E8%BE%91%E7%9A%84%E8%A1%A8%E6%A0%BC

### 重现步骤

![image](https://github.com/Tencent/tdesign-vue-next/assets/150098837/b079657e-a637-4c86-aafb-1ccdd57c475a)

```[tasklist]
### Tasks
- [ ] https://github.com/Tencent/tdesign-vue-next/issues/3563
```

**Comment by chaishi** [comment]
fixed in https://github.com/Tencent/tdesign-vue-next/pull/3577

**Comment by chaishi** [comment]
resolved at `v1.6.8` 2023-11-07
